apps/src/templates/teacherNavigation/TeacherNavigationRouter.tsx

```diff
@@ -302,20 +313,18 @@ const TeacherNavigationRouter: React.FC<TeacherNavigationRouterProps> = ({
                 />
               }
             />
-            {showAITutorTab && (
-              <Route
-                path={TEACHER_NAVIGATION_PATHS.aiTutorChatMessages}
-                element={
-                  <ElementOrEmptyPage
-                    showNoStudents={studentCount === 0}
-                    showNoCurriculumAssigned={!anyStudentHasProgress}
-                    element={applyV1TeacherDashboardWidth(
-                      <TutorTab sectionId={sectionId || 0} />
-                    )}
-                  />
-                }
-              />
-            )}
+            <Route
+              path={TEACHER_NAVIGATION_PATHS.aiTutorChatMessages}
+              element={
+                <ElementOrEmptyPage
+                  showNoStudents={studentCount === 0}
+                  showNoCurriculumAssigned={false}
+                  element={applyV1TeacherDashboardWidth(
+                    <TutorTab sectionId={sectionId || 0} />
+                  )}
```

Comment:
```suggestion
{showAITutorTab ?
                <ElementOrEmptyPage
                  showNoStudents={studentCount === 0}
                  showNoCurriculumAssigned={false}
                  element={applyV1TeacherDashboardWidth(
                    <TutorTab sectionId={sectionId || 0} />
                  )}
                />
              : 
              <Navigate
                  to={TEACHER_NAVIGATION_PATHS.progress}
                  replace={true}
                />
                }
```
What do you think about this kind of logic instead? Then it matches the redirects we already have.